src/libsyntax/ext/tt/quoted.rs

```diff
@@ -44,33 +44,33 @@ impl Delimited {
 }
 
 #[derive(Clone, PartialEq, RustcEncodable, RustcDecodable, Debug)]
-pub struct SequenceRepetition {
+crate struct SequenceRepetition {
     /// The sequence of token trees
-    pub tts: Vec<TokenTree>,
+    crate tts: Vec<TokenTree>,
```

Comment:
Sigh, `crate` visibility looks bad in every position except for perhaps `crate fn`.

What I've been doing is just writing `pub` on fields if it's immediately visible that the struct itself is `pub(crate)`, that already makes sure that the struct (and therefore its fields) will never leave the crate.

Reply:
wouldn't that trigger "unreachable pub" in the future?

Reply:
Looks like it does trigger `unreachable_pub` now.
Well, at least it's not enabled by default.

Reply:
anyway, this become private in the future commit :)